src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -52,7 +52,7 @@
     <CoverletOutput>$([MSBuild]::EnsureTrailingSlash('$(CoverletOutputDirectory)'))$(CoverletOutputName).xml</CoverletOutput>
     <CoverageOutputFilePath>$(CoverletOutput)</CoverageOutputFilePath>
     <CoverageInputFilter>*.coverlet.xml</CoverageInputFilter>
-    <ExcludeByFile Condition="$(ExcludeByFile) == ''"></ExcludeByFile>
+    <ExcludeByFile Condition="$(ExcludeByFile) == ''">$(SourceDir)Common/src/System/SR.*</ExcludeByFile>
```

Comment:
This is the same exclusion used by OpenCover. I didn't see the other SR.cs show up in the projects that I tested, that said, let me look that may be worth considering.

Reply:
You may look at https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets#L13 to know how you can get the path of such resource files. 

Side question, I assume we don't want to see the coverage of the resources. is this true? would it make sense to enable including the resources in the coverage even with some property?

Reply:
The SR.cs for the project is generated from strings.resx and its coverage indicates which messages from string.resx were actually used in the test, I think we should keep as it is.

Coverage numbers will show for type, excluding a specific file makes any sequence point landing on that file to not count in the "total code" of the type. SR is a partial type which projects using resources have their own partial definition. We exclude .../common/src/SR.cs so that part of the type doesn't affect the coverage number of the SR.cs of the project itself, see https://ci.dot.net/job/dotnet_corefx/job/master/job/code_coverage_windows/Code_Coverage_Report/System.IO.Pipes_SR.htm as an example.

Reply:
We already get the path to the SR.cs of the project via the pdb.

Reply:
> The SR.cs for the project is generated from strings.resx and its coverage indicates which messages from string.resx were actually used in the test, I think we should keep as it is.

I agree, but my question is, why we are excluding SR.cs in the common folder. that is what I don't fully get. but this is not big deal anyway.

Reply:
If the project doesn't use all the code paths on Common/src/System/SR.cs then the project never can get to 100% coverage for the SR type, excluding Common/src/System/SR.cs from the coverage calculation for the type ensures that the numbers for SR reflect only the usage of the resources of the project.